lib/board.rb

```diff
@@ -0,0 +1,67 @@
+class Board
+	
+	attr_accessor :cells
+	
+
+	
+	def reset! 			
+		cells.clear
+		self.cells = Array.new(9, " ")
+	
+	end
+
+	def initialize		
```

Comment:
I would just call `self.reset!` here and delegate to that method to create the default empty board.